docs/src/technik/get_started/index.md

```diff
@@ -4,4 +4,16 @@
 
 Im Projekte verwenden wir `checkstyle` und `spotless` um für einen möglichst einheitlichen Codestyle zu sorgen.
 Dazu haben wir Regeln definiert. Diese Regeln und deren Hinterlegung in der jeweiligen IDE ist
-[hier](https://github.com/it-at-m/itm-java-codeformat) beschrieben.
\ No newline at end of file
+[hier](https://github.com/it-at-m/itm-java-codeformat) beschrieben.
+
+## Workflows
+
+### Pull-Requests
+
+Mit der Erstellung eines Pull-Requests wird mittel Workflow geprüft das Code funktional ist: `mvn verify`. Das bedeutet
```

Comment:
"mittels"

Reply:
und "dass Code"

Reply:
fixed in [652151c](https://github.com/it-at-m/Wahllokalsystem/pull/72/commits/652151c80ca46c6943c3daf7dddba1a169451759)